docs/osquery/osquery.asciidoc

```diff
@@ -0,0 +1,394 @@
+[chapter]
+[role="xpack"]
+[[osquery]]
+= Osquery
+
+Osquery augments security, compliance, and operations use cases.
+For example,
+you might use Qsquery to find
+unexpected changes in disk free space or memory usage. Or you might use it to
+improve visibility into active accounts and processes.
+
+Use Osquery UI in {kib} to search, analyze, and visualize Osquery data ingested in {es}:
+
+ * Run live queries with one or more agents
+ * View a history of past queries and their results
+ * Schedule queries to capture changes to OS state over time
+ * Save queries and build a library of queries for specific use cases
+
+The UI is powered by the Osquery Manager integration.
+For information on how to set up Osquery Manager, refer to <<manage-osquery-integrations>>.
+
+[float]
+== Required privileges
+
+To use *Osquery Manager*, you must be assigned to a role with the following privileges:
+
+* `Read` privileges for the `logs-osquery_manager.result*` index.
+* {kib} privileges for **Osquery Manager**. The `All` privilege
+enables you to run, schedule, and save queries. `Read` enables you to
+view live and scheduled query results, but you can run live queries or edit anything.
+
+[float]
+[[osquery-run-query]]
+==  Run live queries
+
+Run a query against one or more agents or policies
+and view the results. This is useful for testing your query and
+troubleshooting problems.
+
+. Open the main menu, and then click *Osquery*.
+. In the *Live queries* view, click **New live query**.
+. Select one or more agents or groups to query. Start typing in the search field,
+and you'll get suggestions for agents by name, ID, platform, and policy.
+. Enter a query or select a query from your saved queries.
++
+[role="screenshot"]
+image::images/enter-query.png[Select saved query dropdown name showing query name and description]
+. Click **Submit**.
+. Review the results in a table, or navigate to *Discover* to dive deeper into the response
+or to *Lens* to build visualizations.
+. To view more information about the request, for example, if there are any failures, open the *Status* tab.
+. When your query captures the data you want, click *Save for later* and define the ID,
+description, and other
+<<osquery-manage-query,details>>.
+. To view a history of the past queries you have run, Open the *Live queries history*.
```

Comment:
I'm not 100% sure this should be step 9 - it's not part of running live queries, but is something you can optionally do later. Could it just be a standalone sentence at the end of the step block?  e.g. end the procedure with step 8, then have a final sentence with this info?